indico/modules/search/controllers.py

```diff
@@ -40,17 +40,24 @@ def _process(self):
 
 
 class RHAPISearch(RH):
+    """API for searching across all records with the current search provider.
+
+    Besides pagination, filters or placeholders may be passed as query parameters.
+    Since `type` may be a list, the results from the search provider are not mixed with
+    the InternalSearch.
+    """
+
     @use_kwargs({
         'page': fields.Int(missing=1),
         'q': fields.String(required=True),
+        'type': fields.List(EnumField(SearchTarget), missing=None)
     }, location='query', unknown=INCLUDE)
-    @use_kwargs({'type': EnumField(SearchTarget)}, location='view_args')
-    def _process(self, type, page, q, **params):
+    def _process(self, page, q, type, internal, **params):
```

Comment:
I don't think this should be there:

```suggestion
    def _process(self, page, q, type, **params):
```